glusterd2/commands/snapshot/snapshot-clone.go

```diff
@@ -105,42 +104,44 @@ func takeSnapshotClone(c transaction.TxnCtx) error {
 func populateCloneBrickMountData(volinfo *volume.Volinfo, name string) (map[string]snapshot.BrickMountData, error) {
 	nodeData := make(map[string]snapshot.BrickMountData)
 
-	brickCount := 0
-	for _, b := range volinfo.GetLocalBricks() {
-		brickCount++
-		mountRoot, err := volume.GetBrickMountRoot(b.Path)
-		if err != nil {
-			return nil, err
-		}
-		mountDir := b.Path[len(mountRoot):]
-		mntInfo, err := volume.GetBrickMountInfo(mountRoot)
-		if err != nil {
-			log.WithError(err).WithField(
-				"brick", b.Path,
-			).Error("Failed to get mount information")
-
-			return nil, err
-		}
-
-		vG, err := lvm.GetVgName(mntInfo.FsName)
-		if err != nil {
-
-			log.WithError(err).WithField(
-				"brick", b.Path,
-			).Error("Failed to get vg name")
-
-			return nil, err
-		}
-		devicePath := fmt.Sprintf("/dev/%s/%s_%s_%d", vG, "clone", name, brickCount)
-
-		nodeID := strings.Replace(b.ID.String(), "-", "", -1)
-
-		nodeData[b.String()] = snapshot.BrickMountData{
-			MountDir:   mountDir,
-			DevicePath: devicePath,
-			FsType:     mntInfo.MntType,
-			MntOpts:    updateMntOps(mntInfo.MntType, mntInfo.MntOpts),
-			Path:       snapshotCloneBrickCreate(volinfo.Name, name, nodeID, mountDir, brickCount),
+	for svIdx, sv := range volinfo.Subvols {
+		for bIdx, b := range sv.Bricks {
```

Comment:
`bIdx` and `svIdx` ca we make `ID` capital

Reply:
I used short forms of `brickIndex` and `subvolIndex`, it is not `ID`

Reply:
sorry I thought its id, then we can ignore it